src/phylum/ci/ci_azure.py

```diff
@@ -164,17 +164,7 @@ def common_ancestor_commit(self) -> Optional[str]:
         remote = git_remote()
 
         if is_in_pr():
-            # There is no single predefined variable available to provide the PR base SHA.
-            # Instead, it can be determined with a `git merge-base` command, like is done for the CINone implementation.
-            # Reference: https://learn.microsoft.com/azure/devops/pipelines/build/variables
-            src_branch = os.getenv("SYSTEM_PULLREQUEST_SOURCEBRANCH", "")
-            tgt_branch = os.getenv("SYSTEM_PULLREQUEST_TARGETBRANCH", "")
-            if not src_branch:
-                raise SystemExit(" [!] The SYSTEM_PULLREQUEST_SOURCEBRANCH environment variable must exist and be set")
-            if not tgt_branch:
-                raise SystemExit(" [!] The SYSTEM_PULLREQUEST_TARGETBRANCH environment variable must exist and be set")
-            print(f" [+] SYSTEM_PULLREQUEST_SOURCEBRANCH: {src_branch}")
-            print(f" [+] SYSTEM_PULLREQUEST_TARGETBRANCH: {tgt_branch}")
+            src_branch, tgt_branch = get_pr_branches()
```

Comment:
Am I assuming correctly that this was extracted into a method because of a new lint of `ruff`? (just curious, I like the change.)

Reply:
Yes, this was done to reduce the [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) of that function from 12 to something under 10...which is the default threshold for the [mccabe rule](https://beta.ruff.rs/docs/rules/complex-structure/). Thankfully, this was the only instance of a function being too complex.